prep-for-module-3-frontend.markdown

```diff
@@ -66,12 +68,13 @@ Create a GitHub gist to answer these questions in as much detail as possible. Im
 * What is React "state?"
 * What does "data down, actions up" mean in React?
 
-_Note:_ As you do your research, stay away from React articles that talk exclusively about "hooks". We _will_ be learning hooks in Mod 3 but they are not a requirement for success in the module.
-
+_Note:_ As you do your research, be aware that if you come across tutorials discussing class components, they may be related to older versions of React. We used to rely on class components before hooks were introduced. We _will_ be learning hooks in Mod 3, and they are now the more modern and recommended approach.  
```

Comment:
I'd love to see us add a note here that its still very possible that they will run into class-based React components in legacy code on the job.